src_docs/components/datagrid/datagrid.component.jsx

```diff
@@ -188,7 +185,6 @@ class DatagridView extends React.Component {
         grid={this.state.gridSettings}
         gridHeader="Example Grid"
         columns={columns}
-        disableActionSave={disableActionSave}
```

Comment:
Now when currency/float/number/text is added to editData on blur, and enabling Save-button cannot require blur. Disabling Save-button did not check if there really was changes, therefore I ended up rather removing disabling than adding new disabling condition.